### DISCUSSION TITLE
How to execute a command upon startup?

### DISCUSSION THREAD

**Discussion by rafaelrinaldi** [LazyVim/LazyVim#4434] [Q&A]
I want to be able to invoke a LazyVim command upon startup.

I have tried to create an autocmd binding it to `VimEnter` and just `echo`ing something, without success. Any help?

**Comment by dpetka2001** [comment]
If you want it to happen at `VimEnter` put the autocmd in `options.lua` instead of `autocmds.lua` I think (not 100% sure).

**Comment by rafaelrinaldi** [comment]
It unfortunately did not work. Tried on both `options.lua` and `autocmds.lua`. Here's the code:

```lua
vim.api.nvim_create_autocmd("VimEnter", {
  callback = function()
    vim.cmd('echo "test"')
  end,
})
```

Any other ideas?

**Reply by dpetka2001** [reply]
Isn't the message present in `:messages`?
